core/integration/module_config_test.go

```diff
@@ -1166,134 +1178,211 @@ func (m *Test) Fn() ([]string, error) {
 	require.Contains(t, strings.TrimSpace(out), "random-file")
 }
 
-const (
-	gitTestRepoURL    = "github.com/dagger/dagger-test-modules"
-	gitTestRepoCommit = "2cb6cb4b0dba52c1e65b3ff46dd1a4a8f9a02f94"
-)
+// VCS to test behavior against
+type vcsTestCase struct {
+	name              string
+	gitTestRepoRef    string
+	gitTestRepoCommit string
+	baseHTMLURL       string // expected base HTML URLs, might differs from ref (e.g. not contain .git)
+	urlPathComponent  string // path separator to access `tree` view of src at commit, per provider
```

Comment:
nit: for readability, if these are part of the "expected" values for the test case, they should be prefixed with `expected`.

Reply:
You're right, this has been implemented 🙏